assets/javascripts/discourse/components/chat-channel-row.js

```diff
@@ -68,13 +66,7 @@ export default Component.extend({
   },
 
   @action
-  leaveChatChannel() {
-    return ajax(`/chat/chat_channels/${this.channel.id}/unfollow`, {
-      method: "POST",
-    })
-      .then(() => {
-        this.chat.stopTrackingChannel(this.channel);
-      })
-      .catch(popupAjaxError);
+  async leaveChatChannel() {
```

Comment:
What does it mean when you make the action async? I see it's used in a button in `chat-channel-row` but I'm not sure what impact adding the modifier would have.

Reply:
it means that you can do : `await leaveChatChannel`